### ISSUE TITLE
Add CLI Argument to prevent update

### ISSUE THREAD

**Issue by hbdermott** [TheRemote/Legendary-Minecraft-Purpur-Geyser#6]
Hey 👋

I'm really liking this docker setup, but I wan't to potentially have different purpur versions. Is there any chance you could add an argument that prevents updates?

**Comment by TheRemote** [comment]
Hello,

So to do this you just request a certain version with -e Version=1.19.3 for example if you wanted to stay on 1.19.3.  You can also request any Purpur version that is up on the servers basically.

In other words you can run any version you'd like.  It just needs to exist on Purpur's servers: https://purpurmc.org/downloads/

For example if you wanted to run 1.19.2 you could do

`-e Version=1.19.2`

This will do exactly what you are saying.  The 1.19.2 version hasn't been updated since November 30th.  None of the old branches really get updates except extremely critical security ones: https://purpurmc.org/downloads/1.19.2

In other words running any version other than the latest essentially is a version freeze.  Hopefully that helps!

**Comment by hbdermott** [comment]
Awesome! My bad for not noticing the environment variable

**Comment by TheRemote** [comment]
Not at all!  I actually had to stop and think for a second when you asked this because I didn't immediately connect the "Version" flag with this either.

I think this will end up being a smart way to do it for you.  It should have the benefit of letting you lock in a version but still receive security updates for the branch.  

For example when the Log4j Java Minecraft exploit came out Paper and Purpur and those teams actually went back and patched their old versions (which normally they don't for older versions like I talked about before but for really big ones they can/have).  

If you had been using the -e Version flag you would actually have received those updates and been protected.  If it was completely frozen then it wouldn't have so that's an example of where I think this could help.

Enjoy and take care!
